Neos.Flow/Classes/ResourceManagement/ResourceRepository.php

```diff
@@ -255,6 +255,33 @@ public function findBySha1AndCollectionName($sha1Hash, $collectionName)
         return $resources;
     }
 
+    /**
+     * Counts all resources with the same SHA1 hash and collection
+     *
+     * @param string $sha1Hash
+     * @param string $collectionName
+     *
+     * @return int
+     */
+    public function countBySha1AndCollectionName($sha1Hash, $collectionName)
```

Comment:
```suggestion
    public function countBySha1AndCollectionName(string $sha1Hash, string $collectionName): int
```
Let's add typehints to newly added methods